src/libs/actions/Policy/Policy.ts

```diff
@@ -529,6 +529,10 @@ function clearXeroErrorField(policyID: string, fieldName: string) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {connections: {xero: {config: {errorFields: {[fieldName]: null}}}}});
 }
 
+function clearSageIntacctErrorField(policyID: string, fieldName: string) {
```

Comment:
`Policy.ts` file is getting bigger and bigger, how about moving this to `connections/SageIntacct.ts`

Reply:
Wouldn't that require refactoring this entire file?